context/AuthContextProvider.tsx

```diff
@@ -65,20 +72,22 @@ export const AuthContextProvider = ({
   };
 
   // get user
-  const getUser = cache(async () => {
+  const getUser = useCallback(async () => {
```

Comment:
Why moving from cache to useCallBack?

Reply:
because cache does not actually work with Next.js.  They have their own way of caching and I was getting a ton of errors in testing cache cause jest did not see it as a "function".  So I looked up next.js caching and that is where the `const export revalidate = 3600` comes from.

Honestly, switching to Appwrite REST API would do much more caching capabilities.

Reply:
useCallback was added cause it kept on calling the function OVER AND OVER AGAIN so I wanted to make sure that it was only called if a dependency changed.  Made the app a bit faster loading.